src/time.rs

```diff
@@ -85,6 +97,12 @@ impl ops::Add<Duration> for Instant {
     }
 }
 
+impl ops::AddAssign<Duration> for Instant {
```

Comment:
I think these should go in a separate commit.

Reply:
:+1: